src/framework/constants.ts

```diff
@@ -0,0 +1,8 @@
+// Players cannot have more than this many cards after the draw phase
+export const MAX_HAND_SIZE = 5;
+
+// Players cannot have more gold than this at any time
+export const MAX_GOLD = 20;
+
+// Upon hitting this worker count ever, player no longer must tech 2 each turn
+export const WORKERS_TO_SKIP_TECH = 10;
```

Comment:
Ironically you aren't using the constants for hand size or workers in the actual impl :P

Reply:
Oh, I thought I was? That's an oversight on my part then.

Reply:
The 79 character limit might have had something to do with it, because I wanted to avoid line wrapping in my tests. But the application code should still use them.